assets/src/dashboard/app/index.js

```diff
@@ -17,30 +17,47 @@
 /**
  * External dependencies
  */
-import { ThemeProvider } from 'styled-components';
+import { StyleSheetManager, ThemeProvider } from 'styled-components';
+import stylisRTLPlugin from 'stylis-plugin-rtl';
+import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
 import theme, { GlobalStyle } from '../theme';
 import KeyboardOnlyOutline from '../utils/keyboardOnlyOutline';
 import { NavigationBar } from '../components';
-import { Route, RouterProvider } from './router';
+import { useRouteHistory, Route, RouterProvider } from './router';
+import { useConfig, ConfigProvider } from './config';
 import { MyStoriesView, TemplatesGalleryView, MyBookmarksView } from './views';
 
-function App() {
+function App({ config }) {
```

Comment:
I've noticed that sometimes there is a space between the function name and params portion and not. Is it variable based on if there are params?

Reply:
I think there‘s currently only a difference between function expressions and declarations. For anything else there shouldn‘t be one. Happy to take a look though 👍